src/Files.App/Controllers/SidebarPinnedController.cs

```diff
@@ -66,25 +69,34 @@ private async Task LoadAsync()
 					{
 						foreach (var item in oldItems)
 						{
-							if (!Model.FavoriteItems.Contains(item))
+							if (Model is not null)
 							{
-								Model.FavoriteItems.Add(item);
+								if (!Model.FavoriteItems.Contains(item))
+								{
+									Model.FavoriteItems.Add(item);
+								}
 							}
 						}
 					}
 					else
 					{
-						Model.AddDefaultItems();
+						Model?.AddDefaultItems();
 					}
 
-					Model.Save();
-					await Model.AddAllItemsToSidebar();
+					Model?.Save();
+					if (Model is not null)
+					{
+						await Model.AddAllItemsToSidebar();
+					}
 					return;
 				}
 				else
 				{
-					Model.AddDefaultItems();
-					await Model.AddAllItemsToSidebar();
+					Model?.AddDefaultItems();
+					if (Model is not null)
+					{
+						await Model.AddAllItemsToSidebar();
+					}					
 					return;
 				}
```

Comment:
Here `Model` is being null-checked too many times. We can achieve the same behaviour by adding `if (Model is null) return;` at line 64.
We're trying not to use curly brackets for single-line `ifs`, can you fix this?